### DISCUSSION TITLE
[@types/react-avatar-editor] Missing Prop

### DISCUSSION THREAD

**Discussion by philinit** [DefinitelyTyped/DefinitelyTyped#61181] [Issues with a @types package]
Missing a prop -  disableHiDPIScaling.  <Boolean>

Package github: https://github.com/mosch/react-avatar-editor#props

[a comment by an automated account is not shown]

**Comment by davidspiess** [comment]
I created a pull request https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61185
